src/components/Checkbox/Checkbox.tsx

```diff
@@ -0,0 +1,61 @@
+import React, {
+  forwardRef,
+  useRef,
+  useImperativeHandle,
+  ChangeEventHandler,
+  ChangeEvent,
+} from 'react';
+import styles from './Checkbox.module.scss';
+
+type CheckboxPropsT = {
+  label?: string;
+  checked?: boolean;
+  disabled?: boolean;
+  classProp?: string;
+  labelClassProp?: string;
+  onChange: Function;
```

Comment:
Could this `Function` type be more specific, to indicate that it receives a single boolean param?